t/spec/with-unicode-multibyte.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "object",
+  "properties": {
+    "foo": {
+      "enum": ["foo–bar"]
```

Comment:
Could you use a heart or something else that is easier to read than "–" ? Example: "json-validater ♥️ utf8"

Reply:
Sure, will change. Happened to be the character I ran into.